toolkit/cli/smart-contracts-commands/src/permissioned_candidates.rs

```diff
@@ -0,0 +1,64 @@
+use crate::PaymentFilePath;
+use ogmios_client::jsonrpsee::client_for_url;
+use partner_chains_cardano_offchain::await_tx::FixedDelayRetries;
+use partner_chains_cardano_offchain::permissioned_candidates::upsert_permissioned_candidates;
+use sidechain_domain::AuraPublicKey;
+use sidechain_domain::GrandpaPublicKey;
+use sidechain_domain::PermissionedCandidateData;
+use sidechain_domain::SidechainPublicKey;
+use sidechain_domain::UtxoId;
+
+#[derive(Clone, Debug, clap::Parser)]
+pub struct UpsertPermissionedCandidatesCmd {
+	#[clap(flatten)]
+	common_arguments: crate::CommonArguments,
+	#[arg(long)]
+	permissioned_candidates_file: String,
```

Comment:
Help is missing here. How is user supposed to know it should be colon separated CSV?